common/src/main/scala/org/apache/celeborn/common/exception/CelebornIOException.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.common.exception
+
+import java.io.IOException
+
+class CelebornIOException(message: String, cause: Throwable)
```

Comment:
How about CelebornException? There might be something like register failed or sendRpc Exception. If you wanna handle celeborn exceptions in spark, I think there is no reason to assume that all exceptions are IO exceptions.

Reply:
> How about CelebornException? There might be something like register failed or sendRpc Exception. If you wanna handle celeborn exceptions in spark, I think there is no reason to assume that all exceptions are IO exceptions.

It should extended from IOException